README.md

```diff
@@ -15,14 +17,20 @@ Before
 </h2>
 <br>
 
-![](inst/assets/README_files/figure-markdown_githubunnamed-chunk-3-1.png)
+<!--html_preserve-->
```

Comment:
The docs should not be updated correctly with images. `html_preserve` didn't work in GitHub.